config_template.cfg

```diff
@@ -1457,7 +1457,7 @@ TABULAR_FORMAT= TECPLOT
 % Files to output 
 % Possible formats : (TECPLOT, TECPLOT_BINARY, SURFACE_TECPLOT,
 %  SURFACE_TECPLOT_BINARY, CSV, SURFACE_CSV, PARAVIEW, PARAVIEW_BINARY, SURFACE_PARAVIEW, 
-%  SURFACE_PARAVIEW_BINARY, MESH, RESTART_BINARY, RESTART_ASCII, CGNS, STL)
+%  SURFACE_PARAVIEW_BINARY, MESH, RESTART_BINARY, RESTART_ASCII, CGNS, STL, CGNS)
```

Comment:
CGNS was already in the list.

Reply:
I am sorry, I didn't notice it.
Removed.